apps/vm/fsw/src/vm_Arming.cpp

```diff
@@ -125,7 +125,6 @@ void VM_Arming::EnteredArmed()
     App.ActuatorArmedMsg.Armed = true;
     App.VehicleControlModeMsg.Armed = true;
     App.SetHomePosition();
-    App.SendVehicleStatusMsg();
```

Comment:
Is this correct?  This means the "SendVehicleStatus" message is only sent once when the state machine transitions to Armed.  We want to trend away from these single one-off messages so we can eventually support checkpoint restart.

Reply:
See line 513 below. This PR removes all calls to SendVehicleStatusMsg() during state changes and sends during cyclic wakeup.

Reply:
Sorry.  I thought this was adding it, not removing it.  Ok.  I get it.  Thanks.